### DISCUSSION TITLE
Bevy won't render

### DISCUSSION THREAD

**Discussion by tylercritchlow** [bevyengine/bevy#10451] [Q&A] [answered]
## Bevy version

0.12.0

## Relevant system information

- Rust version: cargo `1.73.0 (9c4383fb5 2023-08-26)`

- operating system
  - Linux Mint 21.2

If your bug is rendering-related, copy the adapter info that appears when you run Bevy.

```ignore
AdapterInfo { name: "Intel(R) UHD Graphics 620", vendor: 32902, device: 22807, device_type: IntegratedGpu, driver: "Intel Corporation", driver_info: "Intel driver", backend: Vulkan }
```

## What you did

When I build and run the project a blank window pops up.
## What went wrong

- I wanted it to render the scene I set up
- nothing renders and the window is blank.
## Additional information

Other information that can be used to further reproduce or isolate the problem.
This commonly includes:

link to the discord thread where I asked for help: [here](https://discord.com/channels/691052431525675048/1171280167453339769)

**Comment by afonsolage** [comment]
This isn't a problem with the Bevy engine itself,  I saw your code, the problem is that the camera is at the same place as the floor (0.0, 0.0, 0.0). 

Try moving the camera, like so:

```rust
fn spawn_camera(mut commands: Commands) {
    let camera = Camera3dBundle {
        transform: Transform::from_xyz(10.0, 10.0, 0.0).looking_at(Vec3::ZERO, Vec3::Y),
        ..default()
    };
    commands.spawn(camera);
}
```

**Comment by tylercritchlow** [comment]
> This isn't a problem with the Bevy engine itself, I saw your code, the problem is that the camera is at the same place as the floor (0.0, 0.0, 0.0).
> 
> Try moving the camera, like so:
> 
> ```rust
> fn spawn_camera(mut commands: Commands) {
>     let camera = Camera3dBundle {
>         transform: Transform::from_xyz(10.0, 10.0, 0.0).looking_at(Vec3::ZERO, Vec3::Y),
>         ..default()
>     };
>     commands.spawn(camera);
> }
> ```

that is not the problem I've tried that, as well as the example projects that Bevy provides.I provided the discord link in the original post so that you could get the most recent info, thanks.

**Comment by TheoDulka** [comment, marked as answer]
Seems you're having this problem: https://github.com/bevyengine/bevy/issues/8037
Try either updating your graphics driver, or changing your wgpu backend to opengl.
